src/content/glossary/index.md

```diff
@@ -87,6 +91,26 @@ A collection of required information (a block header) about the comprised [trans
   Blocks
 </DocLink>
 
+### block(chain) explorer
+
+A website that allows easy searching and extraction of data from the blockchain.
+
+### block header {#block-header}
+
+The data in a block which is unique to its content and the circumstances in which it was created. It includes the hash of the previous block’s header, the version of the software the block is mined with, the timestamp and the merkle root hash of the contents of the block.
+
+### block propagation {#block-propagation}
+
+The process of transmitting a confirmed block to all other nodes in the network.
+
+### block time
+
+The average time interval between the mining of two blocks. After the merge the block time will be the time between two Beacon blocks (under optimal conditions there will be one block per slot, which occurs every 6 seconds).
```

Comment:
> After the merge the block time will be the time between two Beacon blocks (under optimal conditions there will be one block per slot, which occurs every 6 seconds).

I don't think we need to explain this

Reply:
ok, changed to:

"The average time interval between blocks being added to the blockchain."